docs/rules/no-restricted-properties.md

```diff
@@ -0,0 +1,73 @@
+# Disallow certain object properties (no-restricted-properties)
```

Comment:
disallow the use of specified object properties (no-restricted-properties)

We just started making main heading of docs consistent (even in capitalization) with rules index.
FYI it will eventually become the same format rule-id: rule description, but that fails the tests right now.